### DISCUSSION TITLE
Views not updating in any type of display but its there in dashboard

### DISCUSSION THREAD

**Discussion by rishilahoti** [yhype/yhype-feedback#22] [Q&A]
hello I am facing an issue regarding the profile counter I cannot view the count in any display counter but can see on yhype dashboard

**Comment by antonkomarev** [comment]
@rishilahoti Counter is invisible (1x1 transparent pixel) on your page. It is an intended decision.

**Reply by rishilahoti** [reply]
i am actually doing this as a badge but its not updating
```
<div align="center">
<a>
    <img src="https://komarev.com/ghpvc/?username=rishilahoti&style=for-the-badge">
</a>
```

**Comment by rishilahoti** [comment]
solved
